code/modules/awaymissions/mission_code/beach.dm

```diff
@@ -27,3 +27,154 @@
 	spawn(1)
 		W.loc = get_step(W, dir)
 	water_timer = addtimer(CALLBACK(src, .proc/drip), water_frequency, TIMER_STOPPABLE)
+
+/turf/simulated/floor/beach/away
```

Comment:
This is really gross--we need a unified beach tile set for everything, but this is about getting beach altered ASAP with as little changes as possible.